src/status_im/ui/components/emoji_thumbnail/preview.cljs

```diff
@@ -6,7 +6,8 @@
 (defn emoji-thumbnail [emoji color size]
   (when-not (string/blank? emoji)
     [react/view (styles/emoji-thumbnail-icon color size)
-     [react/text {:style (styles/emoji-thumbnail-icon-text size)} emoji]]))
+     [react/text {:style               (styles/emoji-thumbnail-icon-text size)
```

Comment:
this part is not related to this issue. it will remove invalid prop type error message from the metro.